appinfo/info.xml

```diff
@@ -22,7 +22,7 @@ Guests users can only access files shared to them and cannot create any files ou
 	<screenshot>https://raw.githubusercontent.com/nextcloud/guests/master/screenshots/settings.png</screenshot>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/guests/master/screenshots/dropdown.png</screenshot>
 	<dependencies>
-		<nextcloud min-version="22" max-version="27" />
+		<nextcloud min-version="22" max-version="26" />
```

Comment:
RIP... this breaks installing the app on master (without `--force`). Is is needed?